### DISCUSSION TITLE
openDTU on Battery schaltet meinen Wechselrichter aus bzw. drosselt auf 0

### DISCUSSION THREAD

**Discussion by Agerter** [hoylabs/OpenDTU-OnBattery#592] [General]
Hallo eine Frage in die Runde.

Ich betreibe einen HM600 an einer Batteriebank um damit Nachts einzuspeisen ,das funktioniert soweit ganz gut.
Ich habe die Leistung was der Wechselrichter  Einspeist mit der opeDTu on battery in Verbindung mit einem Shelly 3EM geregelt, das hat auch prim funktioniert.
Jetzt habe ich die Batterie nach ein paar Wochen wieder in Betrieb genommen und die DTU schaltet mir sofort den Wechselrichter aus bez. Drosselt den auf 0 , im Fehlerprotokoll steht über Fernbedienung ausgeschaltet.
Ich habe nichts verändert oder sonst was.

Kann mir einer weiterhelfen warum die DTU den HM nicht mehr drosselt entsprechend des Verbrauchs? 
Ich steh vor einem Rätzel

Vielen Dank für eure Hilfe

**Comment by spcqike** [comment]
Hallo 

Um helfen zu können braucht es noch ein paar Infos.  
was steht denn im log der openDTU? Was hast du im powerlimiter alles eingestellt? Welche Spannungen liegen am DC Eingang an? Bekommst du SoC Werte der Batterie?

**Comment by schlimmchen** [comment]
> Um helfen zu können braucht es noch ein paar Infos. was steht denn im log der openDTU?

Korrekt. Bitte Verbose Logging beim DPL einschalten und die Ausgaben der Konsole teilen.

**Comment by bikerbiker12** [comment]
Ich klinke mich hier mit dem gleichen Problem ein!
Konfig:
HMS-1600 4T
2,5Kw Akku der an alle 4 Eingänge des HMS angeschlossen ist
Akku voll
PV Leistung aktuell 500W
Bezug Netz mit Shelly 3EM wird sauber ausgelesen.
SOC Start bei 30%
SOC Stop bei 15%
Pass Through bei 20%
Die Werte sind bewusst so niedrig um zu testen wann wieder eingespeist wird.

Nach WR Neustart wird kurz eingespeist, dann wird er ausgeschaltet (Gelb). Bin jetzt aktuell zurück auf FW 17.11.23. Das Problem trat schon mehrfach auf. 

[Victron SmartShunt] Text Event H1: Value: -90155
13:23:21.233 > [Victron SmartShunt] Text Event H2: Value: 0
13:23:21.271 > [Victron SmartShunt] Text Event H3: Value: -85435
13:23:21.324 > [Victron SmartShunt] Text Event H4: Value: 5
13:23:21.366 > [Victron SmartShunt] Text Event H5: Value: 0
13:23:21.404 > [Victron SmartShunt] Text Event H6: Value: -720013
13:23:21.442 > [Victron SmartShunt] Text Event H7: Value: 6114
13:23:21.483 > [Victron SmartShunt] Text Event H8: Value: 31511
13:23:21.530 > [Victron SmartShunt] Text Event H9: Value: 0
13:23:21.579 > [Victron SmartShunt] Text Event H10: Value: 1
13:23:21.631 > [Victron SmartShunt] Text Event H11: Value: 0
13:23:21.677 > [Victron SmartShunt] Text Event H12: Value: 0
13:23:21.711 > [Victron SmartShunt] Text Event H15: Value: 0
13:23:21.750 > [Victron SmartShunt] Text Event H16: Value: 0
13:23:21.791 > [Victron SmartShunt] Text Event H17: Value: 1855
13:23:21.836 > [Victron SmartShunt] Text Event H18: Value: 2296
13:23:21.872 > [VE.Direct] serial input (144 Bytes):
13:23:21.910 > [VE.Direct] 0d 0a 48 31 09 2d 39 30 31 35 35 0d 0a 48 32 09
13:23:21.950 > [VE.Direct] 30 0d 0a 48 33 09 2d 38 35 34 33 35 0d 0a 48 34
13:23:22.009 > [VE.Direct] 09 35 0d 0a 48 35 09 30 0d 0a 48 36 09 2d 37 32
13:23:22.070 > [VE.Direct] 30 30 31 33 0d 0a 48 37 09 36 31 31 34 0d 0a 48
13:23:22.115 > [VE.Direct] 38 09 33 31 35 31 31 0d 0a 48 39 09 30 0d 0a 48
13:23:22.155 > [VE.Direct] 31 30 09 31 0d 0a 48 31 31 09 30 0d 0a 48 31 32
13:23:22.194 > [VE.Direct] 09 30 0d 0a 48 31 35 09 30 0d 0a 48 31 36 09 30
13:23:22.265 > [VE.Direct] 0d 0a 48 31 37 09 31 38 35 35 0d 0a 48 31 38 09
13:23:22.302 > [VE.Direct] 32 32 39 36 0d 0a 43 68 65 63 6b 73 75 6d 09 a8
13:23:22.358 > [DPL::loop] ******************* ENTER **********************
13:23:22.403 > [DPL::loop] battery interface enabled, SoC: 100 %, StartTH: 30 %, StopTH: 15 %, SoC age: 0 s
13:23:22.473 > [DPL::loop] dcVoltage: 28.90 V, loadCorrectedVoltage: 28.90 V, StartTH: 26.00 V, StopTH: 25.70 V
13:23:22.510 > [DPL::loop] StartTH reached: yes, StopTH reached: no, inverter is NOT producing
13:23:22.582 > [DPL::loop] SolarPT enabled, Drain Strategy: 0, canUseDirectSolarPower: no
13:23:22.619 > [DPL::loop] battery discharging allowed, PowerMeter: 353 W, target consumption: -15 W
13:23:22.668 > [DPL::setNewPowerLimit] requested: 368 W, last limit: 371 W, diff: 3 W, hysteresis: 15 W, age: 21333 ms
13:23:22.712 > [DPL::loop] ******************* Leaving PL, calculated limit: 368 W, requested limit: 371 W (kept last requested)
13:23:22.761 > [Victron MPPT] Text Event PID: Value: 0XA06B
13:23:22.803 > [Victron MPPT] Text Event FWE: Value: 313FF
13:23:22.842 > [Victron MPPT] Text Event SER: Value: HQ2231MJAJF
13:23:22.880 > [Victron MPPT] Text Event V: Value: 28790
13:23:22.918 > [Victron MPPT] Text Event I: Value: 5900
13:23:22.976 > [Victron MPPT] Text Event VPV: Value: 73960
13:23:23.011 > [Victron MPPT] Text Event PPV: Value: 176
13:23:23.052 > [Victron MPPT] Text Event MPPT: Value: 1
13:23:23.094 > [Victron MPPT] Text Event CS: Value: 4
13:23:23.138 > [Victron MPPT] Text Event OR: Value: 0X00000000
13:23:23.186 > [VE.Direct] serial input (283 Bytes):
13:23:23.231 > [VE.Direct] 3a 41 30 31 30 32 30 30 30 34 34 34 0a 3a 41 44
13:23:23.277 > [VE.Direct] 37 45 44 30 30 33 42 30 30 34 43 0a 3a 41 38 46
13:23:23.319 > [VE.Direct] 45 44 30 30 33 42 30 30 39 34 0a 3a 41 38 45 45
13:23:23.368 > [VE.Direct] 44 30 30 41 43 30 30 32 34 0a 3a 41 44 42 45 44
13:23:23.407 > [VE.Direct] 30 30 44 43 30 35 41 32 0a 3a 41 42 43 45 44 30
13:23:23.441 > [VE.Direct] 30 44 33 34 34 30 30 30 30 38 42 0a 3a 41 42 42
13:23:23.485 > [VE.Direct] 45 44 30 30 45 34 31 43 41 33 0a 3a 41 30 32 30
13:23:23.522 > [VE.Direct] 32 30 30 30 32 30 30 30 30 30 30 34 35 0a 3a 41
13:23:23.563 > [VE.Direct] 32 30 30 31 30 30 38 30 33 33 30 38 30 30 36 46
13:23:23.602 > [VE.Direct] 0a 3a 41 32 37 32 30 30 30 43 30 34 34 30 30 30
13:23:23.642 > [VE.Direct] 30 30 30 0a 0d 0a 50 49 44 09 30 78 41 30 36 42
13:23:23.682 > [VE.Direct] 0d 0a 46 57 45 09 33 31 33 46 46 0d 0a 53 45 52
13:23:23.720 > [VE.Direct] 23 09 48 51 32 32 33 31 4d 4a 41 4a 46 0d 0a 56
13:23:23.791 > [VE.Direct] 09 32 38 37 39 30 0d 0a 49 09 35 39 30 30 0d 0a
13:23:23.896 > [VE.Direct] 56 50 56 09 37 33 39 36 30 0d 0a 50 50 56 09 31
13:23:23.931 > [VE.Direct] 37 36 0d 0a 4d 50 50 54 09 31 0d 0a 43 53 09 34
13:23:23.972 > [VE.Direct] 0d 0a 4f 52 09 30 78 30 30 30 30 30 30 30 30 0d
13:23:24.011 > [VE.Direct] 0a 43 68 65 63 6b 73 75 6d 09 0f
13:23:24.052 > [Victron MPPT] Text Event ERR: Value: 0
13:23:24.095 > [Victron MPPT] Text Event LOAD: Value: ON
13:23:24.135 > [Victron MPPT] Text Event RELAY: Value: OFF
13:23:24.187 > [Victron MPPT] Text Event H19: Value: 2280
13:23:24.233 > [Victron MPPT] Text Event H20: Value: 259
13:23:24.274 > [Victron MPPT] Text Event H21: Value: 963
13:23:24.324 > [Victron MPPT] Text Event H22: Value: 138
13:23:24.359 > [Victron MPPT] Text Event H23: Value: 964
13:23:24.421 > [Victron MPPT] Text Event HSDS: Value: 23
13:23:24.468 > [VE.Direct] serial input (122 Bytes):
13:23:24.522 > [VE.Direct] 3a 41 30 30 30 32 30 30 30 31 34 38 0a 3a 41 42
13:23:24.564 > [VE.Direct] 44 45 44 30 30 31 38 30 30 38 39 0a 0d 0a 45 52
13:23:24.611 > [VE.Direct] 52 09 30 0d 0a 4c 4f 41 44 09 4f 4e 0d 0a 52 65
13:23:24.652 > [VE.Direct] 6c 61 79 09 4f 46 46 0d 0a 48 31 39 09 32 32 38
13:23:24.710 > [VE.Direct] 30 0d 0a 48 32 30 09 32 35 39 0d 0a 48 32 31 09
13:23:24.754 > [VE.Direct] 39 36 33 0d 0a 48 32 32 09 31 33 38 0d 0a 48 32
13:23:24.806 > [VE.Direct] 33 09 39 36 34 0d 0a 48 53 44 53 09 32 33 0d 0a
13:23:24.848 > [VE.Direct] 43 68 65 63 6b 73 75 6d 09 9b
13:23:24.892 > Websocket: [/livedata][3] disconnect
13:23:24.942 > Websocket: [/vedirectlivedata][3] disconnect
13:23:24.980 > Websocket: [/batterylivedata][3] disconnect
13:23:25.031 > [Victron SmartShunt] Text Event PID: Value: 0XA389
13:23:25.075 > [Victron SmartShunt] Text Event V: Value: 28788
13:23:25.126 > [Victron SmartShunt] Text Event I: Value: 6374
13:23:25.163 > [Victron SmartShunt] Text Event P: Value: 184
13:23:25.334 > [Victron SmartShunt] Text Event CE: Value: 0
13:23:25.386 > [Victron SmartShunt] Text Event SOC: Value: 1000
13:23:25.518 > [Victron SmartShunt] Text Event TTG: Value: -1
13:23:25.570 > [Victron SmartShunt] Text Event ALARM: Value: OFF
13:23:25.612 > [Victron SmartShunt] Text Event AR: Value: 0
13:23:25.650 > [Victron SmartShunt] Text Event BMV: Value: SMARTSHUNT 500A/50MV
13:23:25.700 > [Victron SmartShunt] Text Event FW: Value: 0414
13:23:25.742 > [Victron SmartShunt] Text Event MON: Value: 0
13:23:25.782 > [VE.Direct] serial input (131 Bytes):
13:23:25.817 > [VE.Direct] 0d 0a 50 49 44 09 30 78 41 33 38 39 0d 0a 56 09
13:23:25.853 > [VE.Direct] 32 38 37 38 38 0d 0a 49 09 36 33 37 34 0d 0a 50
13:23:25.895 > [VE.Direct] 09 31 38 34 0d 0a 43 45 09 30 0d 0a 53 4f 43 09
13:23:25.939 > [VE.Direct] 31 30 30 30 0d 0a 54 54 47 09 2d 31 0d 0a 41 6c
13:23:25.991 > [VE.Direct] 61 72 6d 09 4f 46 46 0d 0a 41 52 09 30 0d 0a 42
13:23:26.040 > [VE.Direct] 4d 56 09 53 6d 61 72 74 53 68 75 6e 74 20 35 30
13:23:26.084 > [VE.Direct] 30 41 2f 35 30 6d 56 0d 0a 46 57 09 30 34 31 34
13:23:26.123 > [VE.Direct] 0d 0a 4d 4f 4e 09 30 0d 0a 43 68 65 63 6b 73 75
13:23:26.161 > [VE.Direct] 6d 09 51
13:23:26.202 > [Victron SmartShunt] Text Event H1: Value: -90155
13:23:26.255 > [Victron SmartShunt] Text Event H2: Value: 0
13:23:26.297 > [Victron SmartShunt] Text Event H3: Value: -85435
13:23:26.368 > [Victron SmartShunt] Text Event H4: Value: 5
13:23:26.417 > [Victron SmartShunt] Text Event H5: Value: 0
13:23:26.456 > [Victron SmartShunt] Text Event H6: Value: -720013
13:23:26.493 > [Victron SmartShunt] Text Event H7: Value: 6114
13:23:26.546 > [Victron SmartShunt] Text Event H8: Value: 31511
13:23:26.585 > [Victron SmartShunt] Text Event H9: Value: 0
13:23:26.620 > [Victron SmartShunt] Text Event H10: Value: 1
13:23:26.659 > [Victron SmartShunt] Text Event H11: Value: 0
13:23:26.713 > [Victron SmartShunt] Text Event H12: Value: 0
13:23:26.761 > [Victron SmartShunt] Text Event H15: Value: 0
13:23:26.798 > [Victron SmartShunt] Text Event H16: Value: 0
13:23:26.855 > [Victron SmartShunt] Text Event H17: Value: 1855
13:23:26.899 > [Victron SmartShunt] Text Event H18: Value: 2296
13:23:26.950 > [VE.Direct] serial input (144 Bytes):
13:23:26.992 > [VE.Direct] 0d 0a 48 31 09 2d 39 30 31 35 35 0d 0a 48 32 09
13:23:27.063 > [VE.Direct] 30 0d 0a 48 33 09 2d 38 35 34 33 35 0d 0a 48 34
13:23:27.118 > [VE.Direct] 09 35 0d 0a 48 35 09 30 0d 0a 48 36 09 2d 37 32
13:23:27.161 > [VE.Direct] 30 30 31 33 0d 0a 48 37 09 36 31 31 34 0d 0a 48
13:23:27.204 > [VE.Direct] 38 09 33 31 35 31 31 0d 0a 48 39 09 30 0d 0a 48
13:23:27.241 > [VE.Direct] 31 30 09 31 0d 0a 48 31 31 09 30 0d 0a 48 31 32
13:23:27.283 > [VE.Direct] 09 30 0d 0a 48 31 35 09 30 0d 0a 48 31 36 09 30
13:23:27.337 > [VE.Direct] 0d 0a 48 31 37 09 31 38 35 35 0d 0a 48 31 38 09
13:23:27.382 > [VE.Direct] 32 32 39 36 0d 0a 43 68 65 63 6b 73 75 6d 09 a8
13:23:27.422 > [Victron MPPT] Text Event PID: Value: 0XA06B
13:23:27.462 > [Victron MPPT] Text Event FWE: Value: 313FF
13:23:27.501 > [Victron MPPT] Text Event SER: Value: HQ2231MJAJF
13:23:27.540 > [Victron MPPT] Text Event V: Value: 28780
13:23:27.597 > [Victron MPPT] Text Event I: Value: 6100
13:23:27.653 > [Victron MPPT] Text Event VPV: Value: 73930
13:23:27.693 > [Victron MPPT] Text Event PPV: Value: 179
13:23:27.729 > [Victron MPPT] Text Event MPPT: Value: 1
13:23:27.774 > [Victron MPPT] Text Event CS: Value: 4
13:23:27.811 > [Victron MPPT] Text Event OR: Value: 0X00000000
13:23:27.849 > [VE.Direct] serial input (152 Bytes):
13:23:27.899 > [VE.Direct] 3a 41 44 37 45 44 30 30 33 43 30 30 34 42 0a 3a
13:23:27.940 > [VE.Direct] 32 37 32 30 30 30 45 43 34 35 30 30 30 30 44 33
13:23:27.985 > [VE.Direct] 0a 0d 0a 50 49 44 09 30 78 41 30 36 42 0d 0a 46
13:23:28.020 > [VE.Direct] 57 45 09 33 31 33 46 46 0d 0a 53 45 52 23 09 48
13:23:28.059 > [VE.Direct] 51 32 32 33 31 4d 4a 41 4a 46 0d 0a 56 09 32 38
13:23:28.107 > [VE.Direct] 37 38 30 0d 0a 49 09 36 31 30 30 0d 0a 56 50 56
13:23:28.149 > [VE.Direct] 09 37 33 39 33 30 0d 0a 50 50 56 09 31 37 39 0d
13:23:28.190 > [VE.Direct] 0a 4d 50 50 54 09 31 0d 0a 43 53 09 34 0d 0a 4f
13:23:28.230 > [VE.Direct] 52 09 30 78 30 30 30 30 30 30 30 30 0d 0a 43 68
13:23:28.275 > [VE.Direct] 65 63 6b 73 75 6d 09 17
13:23:28.307 > [Victron SmartShunt] Text Event PID: Value: 0XA389
13:23:28.350 > [Victron SmartShunt] Text Event V: Value: 28788
13:23:28.394 > [Victron SmartShunt] Text Event I: Value: 6521
13:23:28.429 > [Victron SmartShunt] Text Event P: Value: 188
13:23:28.471 > [Victron SmartShunt] Text Event CE: Value: 0
13:23:28.509 > [Victron SmartShunt] Text Event SOC: Value: 1000
13:23:28.553 > [Victron SmartShunt] Text Event TTG: Value: -1
13:23:28.592 > [Victron SmartShunt] Text Event ALARM: Value: OFF
13:23:28.631 > [Victron SmartShunt] Text Event AR: Value: 0
13:23:28.669 > [Victron MPPT] Text Event ERR: Value: 0
13:23:28.717 > [Victron MPPT] Text Event LOAD: Value: ON
13:23:28.760 > [Victron MPPT] Text Event RELAY: Value: OFF
13:23:28.800 > [Victron MPPT] Text Event H19: Value: 2280
13:23:28.841 > [Victron MPPT] Text Event H20: Value: 259
13:23:28.877 > [Victron MPPT] Text Event H21: Value: 963
13:23:29.017 > [Victron MPPT] Text Event H22: Value: 138
13:23:29.061 > [Victron MPPT] Text Event H23: Value: 964
13:23:29.111 > [Victron MPPT] Text Event HSDS: Value: 23
13:23:29.159 > [VE.Direct] serial input (94 Bytes):
13:23:29.204 > [VE.Direct] 0d 0a 45 52 52 09 30 0d 0a 4c 4f 41 44 09 4f 4e
13:23:29.241 > [VE.Direct] 0d 0a 52 65 6c 61 79 09 4f 46 46 0d 0a 48 31 39
13:23:29.447 > [VE.Direct] 09 32 32 38 30 0d 0a 48 32 30 09 32 35 39 0d 0a
13:23:29.518 > [VE.Direct] 48 32 31 09 39 36 33 0d 0a 48 32 32 09 31 33 38
13:23:29.559 > [VE.Direct] 0d 0a 48 32 33 09 39 36 34 0d 0a 48 53 44 53 09
13:23:29.601 > [VE.Direct] 32 33 0d 0a 43 68 65 63 6b 73 75 6d 09 9b
13:23:29.638 > [Victron SmartShunt] Text Event BMV: Value: SMARTSHUNT 500A/50MV
13:23:29.679 > [Victron SmartShunt] Text Event FW: Value: 0414
13:23:29.737 > [Victron SmartShunt] Text Event MON: Value: 0
13:23:29.806 > [VE.Direct] serial input (131 Bytes):
13:23:29.847 > [VE.Direct] 0d 0a 50 49 44 09 30 78 41 33 38 39 0d 0a 56 09
13:23:29.887 > [VE.Direct] 32 38 37 38 38 0d 0a 49 09 36 35 32 31 0d 0a 50
13:23:29.930 > [VE.Direct] 09 31 38 38 0d 0a 43 45 09 30 0d 0a 53 4f 43 09
13:23:29.970 > [VE.Direct] 31 30 30 30 0d 0a 54 54 47 09 2d 31 0d 0a 41 6c
13:23:30.573 > [VE.Direct] 61 72 6d 09 4f 46 46 0d 0a 41 52 09 30 0d 0a 42
13:23:30.626 > [VE.Direct] 4d 56 09 53 6d 61 72 74 53 68 75 6e 74 20 35 30
13:23:30.675 > [VE.Direct] 30 41 2f 35 30 6d 56 0d 0a 46 57 09 30 34 31 34
13:23:30.719 > [VE.Direct] 0d 0a 4d 4f 4e 09 30 0d 0a 43 68 65 63 6b 73 75
13:23:30.759 > [VE.Direct] 6d 09 53
13:23:30.800 > [DPL::loop] ******************* ENTER **********************
13:23:30.849 > [DPL::loop] battery interface enabled, SoC: 100 %, StartTH: 30 %, StopTH: 15 %, SoC age: 0 s
13:23:30.920 > [DPL::loop] dcVoltage: 28.90 V, loadCorrectedVoltage: 28.90 V, StartTH: 26.00 V, StopTH: 25.70 V
13:23:31.086 > [DPL::loop] StartTH reached: yes, StopTH reached: no, inverter is NOT producing
13:23:31.133 > [DPL::loop] SolarPT enabled, Drain Strategy: 0, canUseDirectSolarPower: no
13:23:31.172 > [DPL::loop] battery discharging allowed, PowerMeter: 353 W, target consumption: -15 W
13:23:31.209 > [DPL::setNewPowerLimit] requested: 368 W, last limit: 371 W, diff: 3 W, hysteresis: 15 W, age: 22550 ms
13:23:31.252 > [DPL::loop] ******************* Leaving PL, calculated limit: 368 W, requested limit: 371 W (kept last requested)
13:23:31.288 > [Victron SmartShunt] Text Event H1: Value: -90155
13:23:31.334 > [Victron SmartShunt] Text Event H2: Value: 0
13:23:31.370 > [Victron SmartShunt] Text Event H3: Value: -85435
13:23:31.721 > [Victron SmartShunt] Text Event H4: Value: 5
13:23:31.943 > [Victron SmartShunt] Text Event H5: Value: 0
13:23:32.028 > [Victron SmartShunt] Text Event H6: Value: -720013
13:23:32.130 > [Victron SmartShunt] Text Event H7: Value: 6114
13:23:32.180 > [Victron SmartShunt] Text Event H8: Value: 31511
13:23:32.229 > [Victron SmartShunt] Text Event H9: Value: 0
13:23:32.285 > [Victron SmartShunt] Text Event H10: Value: 1
13:23:32.319 > [Victron SmartShunt] Text Event H11: Value: 0
13:23:32.362 > [Victron SmartShunt] Text Event H12: Value: 0
13:23:32.402 > [Victron SmartShunt] Text Event H15: Value: 0
13:23:32.438 > [Victron SmartShunt] Text Event H16: Value: 0
13:23:32.481 > [Victron SmartShunt] Text Event H17: Value: 1855
13:23:32.521 > [Victron SmartShunt] Text Event H18: Value: 2296
13:23:32.560 > [VE.Direct] serial input (144 Bytes):
13:23:32.606 > [VE.Direct] 0d 0a 48 31 09 2d 39 30 31 35 35 0d 0a 48 32 09
13:23:32.640 > [VE.Direct] 30 0d 0a 48 33 09 2d 38 35 34 33 35 0d 0a 48 34
13:23:32.686 > [VE.Direct] 09 35 0d 0a 48 35 09 30 0d 0a 48 36 09 2d 37 32
13:23:32.720 > [VE.Direct] 30 30 31 33 0d 0a 48 37 09 36 31 31 34 0d 0a 48
13:23:32.761 > [VE.Direct] 38 09 33 31 35 31 31 0d 0a 48 39 09 30 0d 0a 48
13:23:32.805 > [VE.Direct] 31 30 09 31 0d 0a 48 31 31 09 30 0d 0a 48 31 32
13:23:32.847 > [VE.Direct] 09 30 0d 0a 48 31 35 09 30 0d 0a 48 31 36 09 30

**Reply by spcqike** [reply]
@bikerbiker12 dein Log zeigt keine Anzeichen, dass der Inverter gestoppt wurde.

gekürzt auf das Wesentliche:

```
13:23:22.358 > [DPL::loop] ******************* ENTER **********************
13:23:22.403 > [DPL::loop] battery interface enabled, SoC: 100 %, StartTH: 30 %, StopTH: 15 %, SoC age: 0 s
13:23:22.473 > [DPL::loop] dcVoltage: 28.90 V, loadCorrectedVoltage: 28.90 V, StartTH: 26.00 V, StopTH: 25.70 V
13:23:22.510 > [DPL::loop] StartTH reached: yes, StopTH reached: no, inverter is NOT producing
13:23:22.582 > [DPL::loop] SolarPT enabled, Drain Strategy: 0, canUseDirectSolarPower: no
13:23:22.619 > [DPL::loop] battery discharging allowed, PowerMeter: 353 W, target consumption: -15 W
13:23:22.668 > [DPL::setNewPowerLimit] requested: 368 W, last limit: 371 W, diff: 3 W, hysteresis: 15 W, age: 21333 ms
13:23:22.712 > [DPL::loop] ******************* Leaving PL, calculated limit: 368 W, requested limit: 371 W (kept last requested)

13:23:30.800 > [DPL::loop] ******************* ENTER **********************
13:23:30.849 > [DPL::loop] battery interface enabled, SoC: 100 %, StartTH: 30 %, StopTH: 15 %, SoC age: 0 s
13:23:30.920 > [DPL::loop] dcVoltage: 28.90 V, loadCorrectedVoltage: 28.90 V, StartTH: 26.00 V, StopTH: 25.70 V
13:23:31.086 > [DPL::loop] StartTH reached: yes, StopTH reached: no, inverter is NOT producing
13:23:31.133 > [DPL::loop] SolarPT enabled, Drain Strategy: 0, canUseDirectSolarPower: no
13:23:31.172 > [DPL::loop] battery discharging allowed, PowerMeter: 353 W, target consumption: -15 W
13:23:31.209 > [DPL::setNewPowerLimit] requested: 368 W, last limit: 371 W, diff: 3 W, hysteresis: 15 W, age: 22550 ms
13:23:31.252 > [DPL::loop] ******************* Leaving PL, calculated limit: 368 W, requested limit: 371 W (kept last requested)
```

sollte er in den 10 Sekunden bei 371W stehen und auch geblieben sein.

Wenn der Inverter von der openDTU-oB gestoppt wird, sollte das im Log ersichtlich sein mit "shutting down" oder sowas.

Nachtrag:

ich sehe grad, es steht ja da ```inverter is NOT producing```. Kannst du einen Log ziehen, wenn er läuft und von selbst ausgeht?

**Reply by bikerbiker12** [reply]
Ok, danke schon mal! Ich kann da nicht so viel rauslesen! 
Ich versuche mal ein Protokoll zu ziehen wie du es beschrieben hast! Wird schwierig, da ich ja nicht immer davor sitzen kann! Ich bekomme jetzt mal ein Push Nachricht wenn er nicht einspeist!

**Reply by Narcodoc** [reply]
Ich habe dasselbe Problem. Ebenfalls ein Hms-1600 4t. Daran hängen 2Paneele über einen B2500 Speicher. (Noch nicht steuerbar über Opendtu)
Tagsüber wird nur ein Panel durchgeschliffen.d.h. einer von 4 Eingängen bekommt Strom.
Der Opendtu regelt runter und der WR geht ins aus.
Strombezug über Hichi Lesekopf ist korrekt.

**Comment by schlimmchen** [comment]
Schalte den Wechselrichter über das Webinterface ein und dann wie spcqike schon schrieb:

> Kannst du einen Log ziehen, wenn er läuft und von selbst ausgeht?

Hast du nur eine OpenDTU/AhoyDTU am laufen, oder funkt da wer dazwischen?

**Reply by bikerbiker12** [reply]
Der WR wird gelb, ganz Off ist er ja dann nicht?!
Es läuft nur die OpenDTU on Battery, mittlerweile mit der Firmware vom November um es zu testen aber mit dem gleichen Problem! Die neue bekomme ich nicht geflasht! 
Mir ist aber heute aufgefallen, dass wenn ich den Wechselrichter vom Netz nehme (Shelly On/off) er wieder normal weiter läuft! Kann aber Zufall gewesen sein! Muss man am GRID Profil eventl was anpassen?! Ich kenn mich da nicht so aus

Danke dir

**Reply by spcqike** [reply]
das GRID Profil musst du nicht anpassen. das geht so oder so nur mit einer originalen DTU in der Cloud, oder?

ein AC On/Off am Shelly führt, wenn ich mich nicht irre, zu einem Neustart des Wechselrichters mit neuer Netzsynchronisation. Da die openDTU(-onBattery) keine persistenten Befehle sendet, sondern nur temporäre, werden diese nicht im Flash des Wechselrichters abgelegt. Nach einem Neustart, startet er also wieder "bei 0", also mit 100% Limit und an/einspeisend.

Ja, gelb bedeutet, dass die openDTU den Wechselrichter per Software aus geschalten hat. Im Log sollte sowas stehen wie "shutdown by remote control". dieser erzwungene Shutdown, davon bräuchte man einen ausführlichen Log, um nachvollziehen zu können, warum die Software den Shutdown-Befehl sendet.

eventuell auch einfach mal Screenshots deiner gesamten Dynamic Power Limiter Einstellungen.

**Reply by Narcodoc** [reply]
Genau das steht bei mir im Log:
Wechselrichter per Fernbedienung ausgeschaltet
Und Limit steht auf unteres Limit
Warum fährt er runter?

**Reply by Narcodoc** [reply]
![Screenshot_20240225-133439.png](https://github.com/helgeerbe/OpenDTU-OnBattery/assets/86588882/afdd9c4f-1dbc-45c1-a173-0f432fd4a0f9)

**Reply by Narcodoc** [reply]
![Screenshot_20240225-133513.png](https://github.com/helgeerbe/OpenDTU-OnBattery/assets/86588882/7006cfa5-0b99-4cce-8daa-6f72f7a7b240)

**Reply by spcqike** [reply]
>  Warum fährt er runter?

@Narcodoc ohne Logauszug ist das schwer zu sagen.
und, hast du eine Batterie und PV Module am Wechselrichter angeschlossen?

**Comment by Agerter** [comment]
Hallo ich hab ein Screenshot der Ereignissanzeige angefügt .Die DTU schaltet sofort den WR aus und dieser ist dann gelb.
Das komische ist wie  ich bereits erwähnt hatte das das ganz ja schon mit der DTU in unveraenderter Form sehr gut funktionierte.
![Screenshot 2024-01-12 175355](https://github.com/helgeerbe/OpenDTU-OnBattery/assets/128357524/0957fbae-8ab5-48cb-9fff-2ac139631f5a)

![Screenshot 2024-01-12 174318](https://github.com/helgeerbe/OpenDTU-OnBattery/assets/128357524/930670e8-cd3b-4b6c-8c5b-f1fe6669469e)

**Comment by Agerter** [comment]
In der Konsole wird mir das hier angezeigt.
![Screenshot 2024-01-12 174720](https://github.com/helgeerbe/OpenDTU-OnBattery/assets/128357524/0ae7f579-674d-44bb-8dc8-a7c02bac262b)

**Reply by schlimmchen** [reply]
Naja, in deinem Fall ist die Antwort recht simpel: Du hast keine Batterieschnittstelle konfiguriert und daher werden die Spannungswerte herangezogen um zu bestimmen, ob die Batterie entladen werden kann. Die Spannung ist aber so (nach deinen eigenen Einstellungen und bei einer 48V Batterie auch nachvollziehbar), dass die Batterie nicht entladen werden darf. Hinzu kommt, dass Solar Passthrough aus ist. Also woher soll die DC-Energie für den Wechselrichter kommen?

[Falls du eine LiFePo4 Batterie mit 16 Zellen hast, dann ist hier dringend angesagt, die Batterie zu laden.]

**Reply by Agerter** [reply]
Ich habe ja keine Batterie an der Schnittstelle angeschlossen.
Ich habe 2 Batteriebänke mit  24V ,diese "simulieren " praktisch die Sonne und werden mit einem Hochsetzsteller auf 42V gesetzt und dann an den PV Eingang angeschlossen.Laden tu ich die per LifePo Ladegeräte und shellys.
Das hat wunderbar funktioniert aber nach dem ich aufgrund Sonnenmangel keine Batterie angeschlossen hatte und jetzt wieder in Betrieb nehmen wollte ging es nicht mehr.
Ich hab jetzt Leistungsbegrenzer ausgeschaltet und habs von Hand auf 50 Prozent gedrosselt da funktioniert es.
![Uploading Screenshot 2024-01-12 194522.png…]()

**Reply by schlimmchen** [reply]
Naja, wenn das so ist, dann sind deine Einstellungen für die Start-Spannung des DPL wohl nicht stimmig mit deinem Setup :man_shrugging: Der Wechselrichter sieht laut deinem Screenshot des Logs 43.6V, aber deine Start-Spannung ist auf 50,0V eingestellt. Also wird der Wechselrichter gestoppt, weil er annimmt die Batterie nicht entladen zu dürfen. Bitte passe die Start-Spannung und die Stop-Spannung in den Einstellungen für den DPL an.

**Reply by Agerter** [reply]
Ok super für die schnelle Antwort das teste ich gleich mal

**Reply by Agerter** [reply]
Hallo danke für die schnelle Antwort ich habe die Startspannung mal auf 24 Volt gesetzt. passiert aber das gleiche. was ich nicht versteh im Prinzip "sieht" die DTU ja gar keine Batterie. Er bekommt 42-43 Volt Eingang von der Batterie (simulierte Sonne) und regelt aufgrund den Leistungsdaten vom 3EM die Leistung des WR . klassische Nulleinspeisung.

**Reply by schlimmchen** [reply]
Falls du gerade nur die Start-Spannung von 50V auf 24V gesetzt hast, und die Stop-Spannung bei 49V belassen hast, dann musst du bitte erstmal die vorhandene Dokumentation genau studieren, und die Einstellungen verstehen.

Andernfalls bitte einen neuen Auszug des Logs schicken.

**Comment by Agerter** [comment]
ICH Habe den Fehler !!! **DC Spannung Start 1 und DC Spannung stopp 0 Eintragen .**... Wer lesen kann ist klar im Vorteil steht ja genau so in der Anleitung

**Comment by TohneH** [comment]
Hi zusammen.
Ich bin absoluter Neuling hier.
Ich habe das gleiche Problem.
Ich konnte die Sendeleistung bei mir nicht erhöhen und somit war ich gezwungen ein Werksreset zu machen.
Anschließend habe ich alle Daten wieder so eingegeben wie vorher.
Aber der Wechselrichter wirdnach kurzer zeit runtergedrosselt und anschließend ausgeschaltet.

Woran kann das liegen?
Ich bin am verzweifeln.
Hat zu diesem Thema jemand eine Lösung???

MfG
TohneH

**Reply by schlimmchen** [reply]
Logs?

**Reply by RickiMcMahon** [reply]
Ich hatte im Power Meter unter -Authorization Type  None
Nachdem ich dort Basic und den admin plus Passwort vom Shelly eingegeben habe funktioniert es bei mir.
